include/pqxx/util.hxx

```diff
@@ -280,18 +280,135 @@ struct PQXX_LIBEXPORT thread_safety_model
 #  define PQXX_POTENTIAL_BINARY_ARG typename
 #endif
 
+struct byte_char_traits : std::char_traits<char>
```

Comment:
I think when a reader is likely to wonder _why_ code does something, that's usually a good place for a comment.  Perhaps you can document this type?  I understand right now why it's there, but if I put myself in the place of someone trying to do maintenance on the library in the future, I think I'd be much happier if this type explained itself.